integration/networking/bridge_linux_test.go

```diff
@@ -1032,10 +937,10 @@ func TestPortMappedHairpin(t *testing.T) {
 	conn.Close()
 
 	const serverNetName = "servernet"
-	network.CreateNoError(ctx, t, c, serverNetName, network.WithDriver(driverName))
+	network.CreateNoError(ctx, t, c, serverNetName)
```

Comment:
Not cricitcal, as `bridge` is the default, but I wonder if we should've kept a `network.WithDriver("bridge")` here; I see we do for various other tests, and it would keep the test closer to what it was before this PR.

Reply:
Yes, 3 of the oldest of 14 test functions in this file explicitly use `WithDriver`.

Because of the split between Linux/Windows tests this PR introduced - this test doesn't need to create a `nat` network any more. So, I removed its `WithDriver` to reduce the clutter and make it more like the other tests. As you say, `bridge` is the default, so there's no need to specify it.

(Unlike those older tests, the newer ones also omit defaults applied by the test helper functions, like `container.WithImage("busybox:latest"), container.WithCmd("top")` in `container.Run`.)

I can put it back if you like? (But I'm not sure it'd be better!) Or, I could put it back, then remove it again in a second commit in this PR?

Reply:
It's probably fine to not update it. My thinking was that existing tests use it;  it's slightly more explicit what we're expecting to test. Also considering if we would ever make the default configurable, but I guess there will be many tests to update in that case, so not critical.

So yeah, not a blocker.